tests/Generated/FailureVerboseArgumentsTest.php.err.txt

```diff
@@ -0,0 +1 @@
+bash: line 1: i_do_not_exist: command not found
```

Comment:
since stdout / stderr are correctly splitted now, this is normal i guess ? should we change something to keep the previous behavior or not ?

Reply:
In fact this would still be displayed in the same output (terminal) so for me it's ok

Reply:
Ok for me too :+1: